src/Core/Utilities/CustomIpRateLimitMiddleware.cs

```diff
@@ -63,28 +64,19 @@ public override Task ReturnQuotaExceededResponse(HttpContext httpContext, RateLi
             {
                 _logger.LogInformation(Constants.BypassFiltersEventId, null,
                     "Request blocked {0}. \nInfo: \n{1}", identity.ClientIp, GetRequestInfo(httpContext));
-                _memoryCache.Set(key, blockedCount,
-                    new MemoryCacheEntryOptions().SetSlidingExpiration(new TimeSpan(0, 5, 0)));
+                _distributedCache.Set(key, blockedCount,
+                    new DistributedCacheEntryOptions().SetSlidingExpiration(new TimeSpan(0, 5, 0)));
             }
         }
 
         private string GetRequestInfo(HttpContext httpContext)
         {
-            if (httpContext == null || httpContext.Request == null)
-            {
-                return null;
-            }
+            if (httpContext == null || httpContext.Request == null) return null;
 
             var s = string.Empty;
-            foreach (var header in httpContext.Request.Headers)
-            {
-                s += $"Header \"{header.Key}\": {header.Value} \n";
-            }
+            foreach (var header in httpContext.Request.Headers) s += $"Header \"{header.Key}\": {header.Value} \n";
 
-            foreach (var query in httpContext.Request.Query)
-            {
-                s += $"Query \"{query.Key}\": {query.Value} \n";
-            }
+            foreach (var query in httpContext.Request.Query) s += $"Query \"{query.Key}\": {query.Value} \n";
```

Comment:
nit: We use curly braces even on single line statements. https://contributing.bitwarden.com/code-style/#control-blocks